__init__.py

```diff
@@ -0,0 +1,44 @@
+import fiftyone.operators as foo
+import fiftyone.operators.types as types
+
+from fiftyone.docs_search.query_index import query_index
+
+
+class SearchDocs(foo.DynamicOperator):
+  def __init__(self):
+    super().__init__(
+      "@voxel51/search-docs",
```

Comment:
Note for anyone curious. This name match match the operator name defined in yaml. As far as the teams runtime is concerned, it will only refer to what is listed in the fiftyone.yml, not that it reads from this file.